NGitLab/Impl/PipelineClient.cs

```diff
@@ -189,5 +189,25 @@ public TestReport GetTestReports(int pipelineId)
         {
             return _api.Get().To<TestReport>($"{_projectPath}/pipelines/{pipelineId.ToStringInvariant()}/test_report");
         }
+
+        public IEnumerable<Bridge> GetBridges(PipelineJobQuery query)
+        {
+            var url = CreateGetBridgesUrl(query);
+            return _api.Get().GetAll<Bridge>(url);
+        }
+
+        public GitLabCollectionResponse<Bridge> GetBridgesAsync(PipelineJobQuery query)
+        {
+            var url = CreateGetBridgesUrl(query);
+            return _api.Get().GetAllAsync<Bridge>(url);
+        }
+
+        private string CreateGetBridgesUrl(PipelineJobQuery query)
+        {
+            var url = $"{_pipelinesPath}/{query.PipelineId}/bridges";
+            url = Utils.AddParameter(url, "scope", query.Scope);
+            url = Utils.AddParameter(url, "include_retried", query.IncludeRetried);
```

Comment:
From the doc, `include_retried` seems not available for bridges query :/

Reply:
Let's make a `PipelineBridgeQuery` then!